### PR TITLE
Fix some missing references.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and `pydocstyle<4` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Anybody can merge after CI pass.

File: doc/api/prev_api_changes/api_changes_3.1.0.rst [hunk 1]
```diff
@@ -236,18 +236,18 @@ Input that consists of multiple empty lists will now return a list of histogram
 values for each one of the lists. For example, an input of ``[[],[]]`` will
 return 2 lists of histogram values. Previously, a single list was returned.
 
-`.backend_bases.Timer.remove_callback` future signature change
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`.backend_bases.TimerBase.remove_callback` future signature change
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (QuLogic) [inline doc/api/prev_api_changes/api_changes_3.1.0.rst:240]: Can you make this code style, not a reference?

File: doc/api/prev_api_changes/api_changes_3.1.0.rst [hunk 2]
```diff
@@ -961,11 +961,11 @@ GUI / backend details
...
 - ``.tk_window_focus``
-- `.backend_gtk3.FileChooserDialog`
-- `.backend_gtk3.NavigationToolbar2GTK3.get_filechooser`
-- `.backend_gtk3.SaveFigureGTK3.get_filechooser`
-- `.NavigationToolbar2QT.adj_window` attribute. This is unused and always ``None``.
-- `.backend_wx.IDLE_DELAY` global variable
+- ``.backend_gtk3.FileChooserDialog``
+- ``.backend_gtk3.NavigationToolbar2GTK3.get_filechooser``
+- ``.backend_gtk3.SaveFigureGTK3.get_filechooser``
+- ``.NavigationToolbar2QT.adj_window`` attribute. This is unused and always ````None````.
```
Comment (QuLogic) [inline doc/api/prev_api_changes/api_changes_3.1.0.rst:967]: Quadrupled ticks on `None` should be reverted.

File: doc/users/dflt_style_changes.rst [hunk 3]
```diff
@@ -597,7 +597,7 @@ in your :file:`matplotlibrc` file.
 ``hexbin``
 ----------
 
-The default value of the ``linecolor`` kwarg for `~matplotlib.Axes.hexbin` has
+The default value of the ``linecolor`` kwarg for `~.Axes.hexbin` has
```
Comment (QuLogic) [inline doc/users/dflt_style_changes.rst:600]:
```suggestion
The default value of the *linecolor* keyword argument for `~.Axes.hexbin` has
```

**Comment by anntzer** [comment]
all done

**Pull request merged**